system_intelligence/gpus_info.py

```diff
@@ -0,0 +1,112 @@
+import typing as t
+from rich import print
+
+from .base_info import BaseInfo
+from .util.unit_conversion_util import bytes_to_hreadable_string, hz_to_hreadable_string
+
+
+class QueryError(RuntimeError):
+    """
+    Indicate that a query failed.
+    """
+
+
+compute_capability_to_architecture = {
+    2: 'Fermi',
+    3: 'Kepler',
+    5: 'Maxwell',
+    6: 'Pascal',
+    7: 'Volta',
+    8: 'Ampere'
```

Comment:
We could and should include the latest GPU generation here.
It requires some googling to get the specs, but absolutely required

Reply:
Will do this!

Reply:
Don't forget it :)

We should also ensure that the case of a user having a GPU that is not (yet) in this dictionary.
It should return a reasonable default aka "Unknown" or something like that